include/deal.II/lac/sparsity_tools.h

```diff
@@ -45,6 +45,17 @@ DEAL_II_NAMESPACE_OPEN
  */
 namespace SparsityTools
 {
+
+  /**
+   * Enumerator with options for partitioner
+   */
+  enum Partitioner
```

Comment:
please use strong enumerators, i.e. `enum class Partitioner`, see https://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum